.test_pull_requests.json.sample

```diff
@@ -60,6 +60,7 @@
       "test_prefix": "Merge Test Results:",
       "pretest_queue_threshold": 2,
       "pretest_settings_key": "test_settings",
+      "preceding_test_settings": "test_settings",
```

Comment:
What you are doing is slightly different than the pretest merge logic.  But the purpose of this key I think is exactly the same.  It says: "I can rely on this other job as a means to tell whether I have been tested already".